README.md

```diff
@@ -1,12 +1,43 @@
 # Trieste
 
-A Bayesian optimization toolbox built on [TensorFlow](https://www.tensorflow.org/). Trieste supports Python 3.7 onwards and uses [semantic versioning](https://semver.org/).
+[![PyPI](https://img.shields.io/pypi/v/trieste.svg)](https://pypi.org/project/trieste)
+[![License](https://img.shields.io/badge/license-Apache-green.svg)](LICENSE)
+[![Quality checks](https://github.com/secondmind-labs/trieste/actions/workflows/quality-checks.yaml/badge.svg)](https://github.com/secondmind-labs/trieste/actions?query=workflows%3Aquality-checks)
+[![Docs](https://github.com/secondmind-labs/trieste/actions/workflows/deploy.yaml/badge.svg)](https://github.com/secondmind-labs/trieste/actions/workflows/deploy.yaml)
+[![Codecov](https://img.shields.io/codecov/c/github/secondmind-labs/trieste/coverage.svg?branch=master)](https://codecov.io/github/secondmind-labs/trieste?branch=master)
+[![Slack Status](https://img.shields.io/badge/slack-trieste-green.svg?logo=Slack)](https://join.slack.com/t/secondmind-labs/shared_invite/zt-ph07nuie-gMlkle__tjvXBay4FNSLkw)
 
-We welcome contributions. See [the guidelines](CONTRIBUTING.md) to get started.
 
-### Installation
+[Documentation (release)](https://secondmind-labs.github.io/trieste/1.0.0/index.html) |
```

Comment:
If you use the landing page it will automatically redirect to the latest release (and we won't accidentally forget to update this):

```suggestion
[Documentation (release)](https://secondmind-labs.github.io/trieste/) |
```